apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -1157,24 +1141,16 @@ module.exports = {
               name: "main-navigation",
               label_singular: "Main Navigation",
               widget: "list",
-              summary: "{{link.content}}",
               fields: [
                 {
-                  label: "Link",
-                  name: "link",
-                  widget: "object",
-                  fields: [
-                    {
-                      label: "Content",
-                      name: "content",
-                      widget: "string",
-                    },
-                    {
-                      label: "Href",
-                      name: "href",
-                      widget: "string",
-                    },
-                  ],
+                  label: "Label",
+                  name: "label",
```

Comment:
```suggestion
                  name: "content",
```
Unlink in Stay in touch where the label is not used anywhere on the site, this one is actually used as content of the link. Both [MUI](https://mui.com/material-ui/api/link/) and [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) refer to it as **content** of component/anchor so I think the original name is the better one.